Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -162,10 +162,8 @@ import PayPalCheckout
         tokenizationClient.tokenize(request: request, returnURL: approval.data.returnURL!.absoluteString) { result in
             switch result {
             case .success(let nonce):
-                self.apiClient.sendAnalyticsEvent("ios.paypal-native.on-approve.succeeded")
                 completion(nonce, nil)
             case .failure(let error):
-                self.apiClient.sendAnalyticsEvent("ios.paypal-native.on-approve.failed")
                 completion(nil, error)
```

Comment:
So to summarize, we want to send all analytic messages from the CheckoutClient rather than the OrderCreation or TokenizeClients set up for specific tasks? The only thing was I didn't know how to set up the tests in case we have specific fail and general fail right after the other. Our tests check for the last analytic message sent.